test/opset_pass/sum_opset1upgrade_pass.cpp

```diff
@@ -0,0 +1,48 @@
+//*****************************************************************************
```

Comment:
We should simplify this filename to match others.

Reply:
Done, name changed to `sum_opset_pass.cpp`